playlist2links

```diff
@@ -17,33 +22,44 @@ if [[ -n $(type -p wget) ]];
 then
 	COMMAND="wget -o /dev/null -O '${TMPFILE}.html' '$URL'"
 fi
-if [[ -z "$COMMAND" ]] && [[ -n $(type -p curl) ]];
+if [ -z "$COMMAND" ] && [[ -n $(type -p curl) ]];
 then
 	COMMAND="curl -s -o '${TMPFILE}.html' '$URL'"
 fi
-if [[ -z "$COMMAND" ]];
+if [ -z "$COMMAND" ];
```

Comment:
It was only out of curiosity, I did not think that quoting was preventing expansion. Regarding your commit, great! I'll resolve the conflict and merge it.